browser/android/background_video/background_video_playback_tab_helper.cc

```diff
@@ -19,17 +19,61 @@
 #include "url/gurl.h"
 
 namespace {
-constexpr char16_t kYoutubeBackgroundPlaybackScript[] =
-    u"(function() {"
-    "    if (document._addEventListener === undefined) {"
-    "        document._addEventListener = document.addEventListener;"
-    "        document.addEventListener = function(a,b,c) {"
-    "            if(a != 'visibilitychange') {"
-    "                document._addEventListener(a,b,c);"
-    "            }"
-    "        };"
-    "    }"
-    "}());";
+constexpr char16_t kYoutubeBackgroundPlaybackAndPipScript[] =
+    uR"(
```

Comment:
we can remove this extra enclosing `()` as well

Reply:
nvm, I see now that it is part of the text delimiter

Reply:
Yes!

```cc
auto str = R"(This is a "raw string"!)";
std::cout << str;
// Output: This is a "raw string"!
```